components/LandingPage/sections/Brands.tsx

```diff
@@ -0,0 +1,100 @@
+import Image, { StaticImageData } from "next/image"
+
+import { useMediaQuery } from "usehooks-ts"
+import ExtendImage from "../../../public/assets/Landing/brands/extend.png"
+import LampImage from "../../../public/assets/Landing/brands/lamp.png"
+import ChainImage from "../../../public/assets/Landing/brands/chain.png"
+import NetworkingImage from "../../../public/assets/Landing/brands/networking.png"
+import Title from "./brands/Title"
+import Desc from "./brands/Desc"
+
+interface ItemData {
+  image: StaticImageData
+  width: number
+  height: number
+  title: React.ReactNode
+  text: React.ReactNode
+  key: string
+}
+
+const Brands = () => {
+  const match490 = useMediaQuery("(max-width: 490px)")
```

Comment:
why are we matching `490` exactly?

could we make this variable name more easy to understand its functionality?

`isMobile` maybe? What is the breakpoint you're checking here?